js/services.js

```diff
@@ -0,0 +1,49 @@
+class TabLink {
+  constructor(link) {
+    this.link = link;
+    // dataset numbers 1 - 4
+    this.linkData = this.link.dataset.tab;
+
+    if (this.linkData === '0') {
+      this.cards = document.querySelector('.content');
+    } else {
+      // content by dataset numbers
+      this.cards = document.querySelectorAll(`.content[data-tab="${this.linkData}"]`);
+    }
+
+    this.cards = Array.from(this.cards).map(cards => new TabCard(cards));
+
+    // event listener that calls method
+    this.link.addEventListener('click', () => {
+      event.stopPropagation();
```

Comment:
You should explicitly pass the event here! This works in Chrome because they do some voodoo for you, but it doesn't work in most other browsers (i.e. Firefox). The fix:
```javascript
 this.link.addEventListener('click', e => {
  e.stopPropagation();
  // ...
```